tapchannel/auf_leaf_signer_test.go

```diff
@@ -207,7 +207,7 @@ func TestAuxLeafSignerCancelAndQuit(t *testing.T) {
 		// Another component could have sent the cancel signal; we'll
 		// send that before the quit signal.
 		close(cancelChan)
-		time.Sleep(time.Millisecond)
+		time.Sleep(time.Microsecond * 250)
```

Comment:
Isn't this the opposite of "relaxing" the timing? Since this is now 0.25 milliseconds as before it was 1 millisecond?

Reply:
What Oli says. :point_up:

Reply:
Can we get rid of this sleep?

Reply:
The wording is sort of flipped; the 'timing' here is how long the aux signer will take to skip the remaining sig jobs. 

Ideally, the last job is never processed after the quit signal is sent. I didn't account for the fact that either case of the quit-cancel switch may fire, and that jobs will get processed a lot faster once one signal is received.

After adding jobs and decreasing the timeout, I can no longer repro this flake.

Reply:
> Can we get rid of this sleep?

Actually, yes - I think there doesn't need to be known delay between the cancel and the quit, it was a leftover from fixing the initial issue with these signals.